spec/tags/language/method_tags.txt

```diff
@@ -1,5 +1,6 @@
 fails:A method assigns local variables from method parameters for definition 'def m() end'
 fails:A method assigns local variables from method parameters for definition 'def m(*a) a end'
+fails:"A method assigns local variables from method parameters for definition \n    def m(a)\n      a.delete(:one); a\n    end"
```

Comment:
(Not Maple's fault or problem.)

I really can't stand these `evaluate` specs, Benoit! Do you think they're a good idea? They were one of the previous maintainer's wacky ideas weren't they?

Reply:
Yeah, they're rather weird and they don't have a proper description.
I think it would be best to just use:
```ruby
it "description" do
  def m(a)
    ...
  end
  m(**h).should ...
end
```